### PR TITLE
Parsing API for non akka-http clients

### LINKED ISSUE #84: Stream API for non akka-http clients
I'm using akka-sse in play framework app. I would be nice to see plain stream API for SSE client, without akka-http-experimental dependency.

Something like this

```
package de.heikoseeberger.akkasse

import akka.stream.scaladsl.Source
import akka.util.ByteString

object ServerSendEventFlow {
  def toEventSource(source:Source[ByteString, Any], maxLineSize: Int = 8192, maxEventSize:Int = 8192) = {
    source
      .via(new LineParser(maxLineSize))
      .via(new ServerSentEventParser(maxEventSize))
  }
}
```

### PR DESCRIPTION
**Pull request by maxcom**
fixes #84

### CODE REVIEW

File: akka-sse/src/main/scala/de/heikoseeberger/akkasse/EventStreamParserFlow.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ * limitations under the License.
+ */
+
+package de.heikoseeberger.akkasse
+
+import akka.NotUsed
+import akka.stream.scaladsl.Flow
+import akka.util.ByteString
+
+object EventStreamParserFlow {
```
Comment (hseeberger) [inline akka-sse/src/main/scala/de/heikoseeberger/akkasse/EventStreamParserFlow.scala]: Please remove the "Flow" suffix from the name.

File: akka-sse/src/main/scala/de/heikoseeberger/akkasse/EventStreamParserFlow.scala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  /**
+   * Flow that converts raw byte string input into ServerSentEvent's.
```
Comment (hseeberger) [inline akka-sse/src/main/scala/de/heikoseeberger/akkasse/EventStreamParserFlow.scala]: s/ServerSentEvent's/[[ServerSentEvent]]s

**Comment by hseeberger** [comment]
Very nice! Please fix the two minor wording issues.

**Comment by maxcom** [comment]
ok

**Comment by maxcom** [comment]
done, please review

**Pull request merged**